Tests/RegressionTests/SVGTests.swift

```diff
@@ -293,7 +293,7 @@ extension WKWebView {
   }
 }
 
-private class WKWebViewSnapshoter {
+class WKWebViewSnapshoter {
```

Comment:
Probably move to Support.swift?

Reply:
It is for SVG so SVGTests seems to be appropriate place, just like with `renderPDF()`

Reply:
`*Tests.swift` files are appropriate for tests and for private helpers that are used in those tests. Code that is shared between different `*Tests.swift` files should be extracted to `Support.swift` or something similar like `PDFTestSupport.swift`. `renderPDF()` must've been `fileprivate` too, my bad.